app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
Code Style: It's good practice to use annotations for function parameters and return types. In this case, `Cafe` is a class, so you should use it as a type hint for the `cafe` parameter. The correct annotation would be `cafe: Cafe`.

Reply:
That's how it's written